manifests/07-downloads-deployment.yaml

```diff
@@ -38,6 +38,10 @@ spec:
           name: http
           protocol: TCP
         command: ["/bin/sh"]
+        resources:
+          requests:
+            cpu: 10m
+            memory: 50Mi
```

Comment:
This seems too low for what the pod actually uses :(

Reply:
![Screen Shot 2019-05-16 at 4 39 34 PM](https://user-images.githubusercontent.com/1167259/57885567-3b7b2900-77f9-11e9-96e4-fb69113b9b30.png)

Reply:
much of that is fs cache.  i suspected that and did a `drop_caches` on the worker running the pod and it dropped to 30Mi.